README.md

```diff
@@ -2,7 +2,24 @@
 
 # New Relic Node.js Examples
 
-This repository will contain examples of agent features.  Browse the folders for different self-contained, working examples.
+This repository contains example application and scripts that demonstrate functionality of the [New Relic Node.js Agent](https://github.com/newrelic/node-newrelic).  
+
+ * [AI Monitoring](./ai-monitoring) - examples using supported 3rd party libraries to produce AI Monitoring(AIM) telemetry
+ * [Logs in Context](./application-logging) - examples using logging libraries to demonstrate agent log decoration and forwarding of logs to New Relic
+ * [Code Level Metrics](./code-level-metrics) - example demonstrating span attributes that help drive the [CodeStream](https://newrelic.com/codestream) integration
+ * [Custom Instrumentation](./custom-instrumentation) - examples demonstrating the use of the [Node.js agent API](https://newrelic.github.io/node-newrelic/API.html)
+ * [Elasticsearch](./elasticsearch) - example demonstrating elasticsearch with Node.js agent
+ * [Error Fingerprinting](./error-fingerprinting) - example using [setErrorCallback](https://newrelic.github.io/node-newrelic/API.html#setErrorGroupCallback) and [noticeError](https://newrelic.github.io/node-newrelic/API.html#noticeError) to [group errors](https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#error-groups) produced in Node.js application
+ * [ESM](./esm-app) - example demonstrating how to load agent in an [ESM](https://nodejs.org/api/esm.html) application. It also demonstrates how to register custom instrumentation for a ESM package.
+ * [GraphQL Dataloader](./graphql-koa-dataloader) - example using Apollo Server, koa and GraphQL dataloader
+ * [Kafkajs](./kafkajs) - example demonstrating [kafkajs](https://kafka.js.org/) with Node.js agent 
+ * [Mock Infinite Tracing Server](./mock-infinite-tracing-server) - mock gRPC server to use to locally test [infinite tracing](https://docs.newrelic.com/docs/distributed-tracing/infinite-tracing/introduction-infinite-tracing/) with a Node.js applciation 
+ * [Nest](./nestjs) - examples demonstrating Nestjs with Node.js agent
+ * [Next.js](./nextjs) - examples demonstrating Next.js with Node.js agent
+ * [Prisma](./prisma-app) - example demonstrating Prisma with Node.js agent
+ * [Sequelize](./sequelize-app) - example demonstrating sequelize with Node.js agent
+ * [Express](./simple-express-app) - example demonstrating express with Node.js agent
```

Comment:
```suggestion
 * [Sequelize](./sequelize-app) - example demonstrating sequelize with the Node.js agent
 * [Express](./simple-express-app) - example demonstrating express with the Node.js agent
```

Reply:
I'd recommend the same change with lines 17, 18, and 19 as well.